config.rb

```diff
@@ -9,19 +9,19 @@
     title: "Teacher Training API documentation",
     pages: GitHubRepoFetcher.instance.docs(
       service_name: "Teacher Training API",
-      repo_name: "DFE-Digital/teacher-training-api",
+      repo_name: "DFE-Digital/publish-teacher-training",
       path_in_repo: "docs",
-      path_prefix: "services/teacher-training-api",
+      path_prefix: "services/publish-teacher-training",
       ignore_files: %w[api.md],
     ),
   },
   {
     title: "Teacher Training API decisions",
     pages: GitHubRepoFetcher.instance.docs(
       service_name: "Teacher Training API",
```

Comment:
Pls rename here and elsewhere to `Publish teacher training courses` and this is good to go!